osutil/user_test.go

```diff
@@ -279,6 +280,82 @@ func (s *createUserSuite) TestIsValidUsername(c *check.C) {
 	}
 }
 
+type delUserSuite struct {
+	mockUserDel *testutil.MockCmd
+	opts        *osutil.DelUserOptions
+	sudoersd    string
+	restore     func()
+}
+
+var _ = check.Suite(&delUserSuite{opts: nil})
+var _ = check.Suite(&delUserSuite{opts: &osutil.DelUserOptions{ExtraUsers: true}})
+
+func (s *delUserSuite) SetUpTest(c *check.C) {
+	s.mockUserDel = testutil.MockCommand(c, "userdel", "")
+	s.sudoersd = c.MkDir()
+	s.restore = osutil.MockSudoersDotD(s.sudoersd)
+}
+
+func (s *delUserSuite) TearDownTest(c *check.C) {
+	s.mockUserDel.Restore()
+	s.restore()
+}
+
+func (s *delUserSuite) expectedCmd(u string) []string {
+	if s.opts != nil && s.opts.ExtraUsers {
+		return []string{"userdel", "--remove", "--extrausers", u}
+	}
+	return []string{"userdel", "--remove", u}
+}
+
+func (s *delUserSuite) TestDelUser(c *check.C) {
+	c.Assert(osutil.DelUser("u1", s.opts), check.IsNil)
```

Comment:
u1 -> blast from the past :) ?

Reply:
before changing it so the opts came from the suite itsef, I was creating two users u1 and u2...

Reply:
before changing it so the opts came from the suite itself, I was creating two users u1 and u2...